Scripts/EnemyAnimatorController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyAnimatorController : MonoBehaviour
+{
+    Animator animator;
+    void Start()
+    {
+        animator = gameObject.GetComponent<Animator>();
```

Comment:
Kendi objesinin üzerindeki component'i almaya çalıştığı için, direkt bu şekilde kullanılabilir.
```suggestion
        animator = GetComponent<Animator>();
```